app/src/main/java/com/app/calculator/Operator.java

```diff
@@ -2,7 +2,8 @@
 
 public enum Operator {
```

Comment:
Consider using Typedef instead of enum, enums are bad for performance.Please take a look at the enum section of this article: https://medium.com/@fernandosierra/android-support-annotations-331d925ab26c